doc-Managing_Providers/topics/Adding_Viewing_Network_Providers.adoc

```diff
@@ -1,13 +1,28 @@
-= Adding or Viewing Network Providers 
+= Adding or Viewing Network Providers
 
 [NOTE]
 ====
-All supported network providers, that is OpenStack Network, Azure Network, and Amazon EC2 Network, are added or removed automatically upon adding or removing the respective cloud provider. 
+All supported network providers - OpenStack Network, Azure Network, and Amazon EC2 Network, are added or removed automatically upon adding or removing the respective cloud provider.
```

Comment:
Tiny editing nitpick: can this be changed to an em dash instead of a hyphen?
"All supported network providers - "